dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py

```diff
@@ -42,5 +42,31 @@ def lease(db, dhcp_interface):
     click.echo(tabulate(table, headers=headers))
 
 
+def count_ipv4(start, end):
+    ip1 = int(ipaddress.IPv4Address(start))
```

Comment:
IPv4 can be represented by a uint32 integer, should we use long instead int here?